lib/integrations/flowdock.ts

```diff
@@ -25,11 +27,15 @@ export class FlowdockIntegration implements Integration {
 		return Promise.resolve();
 	}
 
-	public async mirror(_card: any, _options: any) {
+	public async mirror(_contract: Contract, _options: { actor: string }) {
 		return [];
 	}
 
-	public async translate(event: any): Promise<any> {
+	// TS-TODO: Use EventContract with properly typed payload
```

Comment:
Should probably go ahead and update the `translate()` function definition in `worker` to expect an `EventContract` instead of a `Contract` and define as much of `data.payload` as possible.